db/data_migration/20170522134843_convert_wlna_to_news_articles.rb

```diff
@@ -0,0 +1,39 @@
+def update_db_records_to_news_article(editions)
+  editions.update_all(
+    type: "NewsArticle",
+    news_article_type_id: NewsArticleType::WorldNewsStory.id
+  )
+end
+
+def update_rummager(editions)
+  editions.each(&:save)
+end
+
+def editions_including_deleted(document_id)
+  Edition.unscoped.where(document_id: document_id)
+end
+
+world_location_news_articles = WorldLocationNewsArticle.all
+wlna_documents = Document.where(id: world_location_news_articles.pluck(:document_id).uniq)
+
+puts "Migrating #{wlna_documents.count} WLNA -> NewsArticle"
+
+wlna_documents.each do |wlna_document|
+  wlna_document.document_type = "NewsArticle"
```

Comment:
The `Document` gets saved when we call `#update_attributes` on line 26. In addition to making the `Document` be the right 'type', setting it here is necessary for the slug to be 'seen' as a duplicate (the unique constraint on `slug` in the DB is scoped to `document_type`).